### PR TITLE
Fix Issue #89

### PR DESCRIPTION
**Pull request by swissonid**
Hi we (thx to @mosberger) found a solution how your lib would work with the newest kotlin version.

### CODE REVIEW

**Comment by hotchemi** [comment]
Thx!

File: sample/build.gradle [hunk 1]
```diff
@@ -22,7 +22,7 @@ android {
 
 dependencies {
     compile "com.android.support:appcompat-v7:$SUPPORT_LIBRARY_VERSION"
-
+    compile "com.android.support:design:$SUPPORT_LIBRARY_VERSION"
```
Comment (hotchemi) [inline sample/build.gradle]: Why is it needed?

Comment (swissonid) [inline sample/build.gradle]: Oh sorry, tried some something out. And just overlooked this line. So you can delete. Sorry again :'(

**Comment by aurae** [comment]
Could you squash your commits into a single contribution? Other than that, this looks good to me.

**Comment by hotchemi** [comment]
@swissonid Could u squash?

**Comment by Dev-iL** [comment]
As mentioned [here](https://github.com/hotchemi/PermissionsDispatcher/commit/0f46f62daf87dc51631f029e5e26e741fb61ef8b#commitcomment-16208384) - the update of Kotlin breaks data binding.

**Comment by hotchemi** [comment]
Thx for your report. I'll  check this issue.

**Pull request merged**